src/Slices/Status/UI/Page.test.tsx

```diff
@@ -228,6 +226,13 @@ test("GIVEN StatusPage with support extension WHEN user click download THEN an a
 });
 
 test("GIVEN StatusPage with support extension WHEN user click download THEN button goes through correct phases", async () => {
+  server.use(
+    http.get("/api/v2/support", async () => {
+      await delay(100);
```

Comment:
Is there a reason why you need the delay?

Reply:
without delay I could not catch state change